backend/inspirehep/mailing/providers/sendgrid.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import structlog
+from flask import current_app
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+from inspirehep.mailing.errors import SendGridMissingAPIToken
+
+LOGGER = structlog.getLogger()
+
+
+def send_email(sender, receiver, subject, content, cc=[]):
```

Comment:
* `recipient` instead of `receiver`
* **Never use mutable default args** (unless you do it on purpose). Now if you call `send_email` twice, the second time `cc` will still contain the content from the first run. Here you can do instead:
```suggestion
def send_email(sender, receiver, subject, content, cc=()):
```